docs/apidocs/objective.rst

```diff
@@ -1,10 +1,12 @@
-.. _objective:
-
-##################
-Objective Function
-##################
+#############################################################
+Objective Function (:mod:`qiskit_addon_aqc_tensor.objective`)
+#############################################################
 
 .. automodule:: qiskit_addon_aqc_tensor.objective
    :no-members:
    :no-inherited-members:
    :no-special-members:
+
+.. currentmodule:: qiskit_addon_aqc_tensor.objective
+
+.. autoclass:: OneMinusFidelity
```

Comment:
This class is tiny

Reply:
Tiny in the number of methods?  It is indeed.  The most important one, actually, is `__call__`, which I could never manage to be rendered by Sphinx.

Reply:
This was later fixed in https://github.com/Qiskit/qiskit-addon-aqc-tensor/pull/33.